nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -72,6 +72,38 @@
 }</programlisting>
     </para>
    </listitem>
+
+   <listitem>
+    <para>
+      The NixOS options <literal>nesting.clone</literal> and
+      <literal>nesting.children</literal> have been deleted, and
+      replaced with named <literal>specialisation</literal>
+      configurations.
+    </para>
+
+    <para>
+      Replace a <literal>nesting.children</literal> entry with:
+
+<programlisting>{
+specialisation.example-sub-configuration = {
+  configuration = {
+    ...
+  };
+};</programlisting>
+
+    </para>
+    <para>
+      Replace a <literal>nesting.clone</literal> entry with:
+
+<programlisting>{
+specialisation.example-sub-configuration = {
+  inheritParentConfig = false;
+  configuration = {
+    ...
+  };
+};</programlisting>
```

Comment:
Aren't the examples wrong way around here? I was under impression `nesting.clone` would be based on the parent configuration, with `nesting.children` starting from a blank slate.

Reply:
Fixed up, good catch :)